src/app/(documentation)/docs/page.tsx

```diff
@@ -1,10 +1,16 @@
-import { Container } from '@/components/elements/container';
-import { Heading } from '@/components/elements/heading';
+import { DocumentationLayout } from '@/components/layout/documentation';
+import { getDocBySlug, ROOT_DOC_SLUG, ROOT_DOCS_DIRECTORY } from '@/lib/mdx/docs';
+import { findFile } from '@/lib/utils/find-file';
+import { getHeadingLinks } from '@/lib/utils/get-heading-links';
 
+// TODO: remove when we can do the redirect at production level
 export default async function DocsPage() {
+  const { title, content: Content } = await getDocBySlug(ROOT_DOC_SLUG);
+  const [filePath] = findFile(ROOT_DOCS_DIRECTORY, ROOT_DOC_SLUG, '.mdx', true);
+  const links = await getHeadingLinks(filePath);
   return (
-    <Container>
-      <Heading color="textGradient">Docs</Heading>
-    </Container>
+    <DocumentationLayout title={title} links={links}>
```

Comment:
Could you check the validation for `NavigationSidebarLinks`? Currently, it is rendering a `0` if no links exist. This a good example of why we should validate the length explicitly `length > 0` if we use `&&` to optional rendering or we can use the ternaries:

Instead of 

```tsx
   {links?.length && <NavigationSidebarLinks title="On this page" links={links} />}
```
Do:

```tsx
{links?.length ? <NavigationSidebarLinks title="On this page" links={links} /> : null}
```
![Screenshot 2024-03-22 at 14 37 05](https://github.com/bejamas/httptoolkit-website/assets/22168189/398f0411-81dc-4fe6-bfbf-aaa7c9a2a8fc)

Reply:
@RBoraurito, could you also change the hover style? Currently, it makes the text bolder, which causes the text to shift.